extra/3-magic-8-ball.js

```diff
@@ -43,10 +43,53 @@
     Very doubtful.
 */
 
+let veryPositiveAnswers = [
+  "It is certain.",
+  "It is decidedly so.",
+  "Without a doubt.",
+  "Yes - definitely.",
+  "You may rely on it.",
+];
+
+let positiveAnswers = [
+  "As I see it, yes.",
+  "Most likely.",
+  "Outlook good.",
+  "Yes.",
+  "Signs point to yes.",
+];
+
+let negativeAnswers = [
+  "Reply hazy, try again.",
+  "Ask again later.",
+  "Better not tell you now.",
+  "Cannot predict now.",
+  "Concentrate and ask again.",
+];
+
+let veryNegativeAnswers = [
+  "Don't count on it.",
+  "My reply is no.",
+  "My sources say no.",
+  "Outlook not so good.",
+  "Very doubtful.",
+];
+
+let allAnswers = [
+  ...veryPositiveAnswers,
+  ...positiveAnswers,
+  ...negativeAnswers,
+  ...veryNegativeAnswers,
+];
+
 // This should log "The ball has shaken!"
+
 // and return the answer.
 function shakeBall() {
   //Write your code in here
+  console.log("The ball has shaken!");
```

Comment:
After the addition of the console.log, I had a mistake TypeError: expect(...).toBeOneOf is not a function.
I added "const { toBeOneOf } = require("jest-extended");" to the test file and solved the problem. Try this out